src/wp-includes/fonts/class-wp-font-library.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Font Library class.
+ *
+ * This file contains the Font Library class definition.
+ *
+ * @package    WordPress
+ * @subpackage Font Library
+ * @since      6.4.0
+ */
+
+/**
+ * Font Library class.
+ *
+ * @since 6.4.0
+ */
+class WP_Font_Library {
+
+	/**
+	 * Provide the expected mime-type value for font files per-PHP release. Due to differences in the values returned these values differ between PHP versions.
+	 *
+	 * This is necessary until a collection of valid mime-types per-file extension can be provided to 'upload_mimes' filter.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param array $php_version_id The version of PHP to provide mime types for. The default is the current PHP version.
+	 *
+	 * @return Array A collection of mime types keyed by file extension.
+	 */
+	public static function get_expected_font_mime_types_per_php_version( $php_version_id = PHP_VERSION_ID ) {
+
+		$php_7_ttf_mime_type = $php_version_id >= 70300 ? 'application/font-sfnt' : 'application/x-font-ttf';
+
+		return array(
+			'otf'   => 'application/vnd.ms-opentype',
+			'ttf'   => $php_version_id >= 70400 ? 'font/sfnt' : $php_7_ttf_mime_type,
+			'woff'  => $php_version_id >= 80100 ? 'font/woff' : 'application/font-woff',
+			'woff2' => $php_version_id >= 80100 ? 'font/woff2' : 'application/font-woff2',
+		);
+	}
+
+	/**
+	 * Font collections.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @var array
+	 */
+	private static $collections = array();
+
+	/**
+	 * Register a new font collection.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param array $config Font collection config options.
+	 *                      See {@see wp_register_font_collection()} for the supported fields.
+	 * @return WP_Font_Collection|WP_Error A font collection is it was registered successfully and a WP_Error otherwise.
+	 */
+	public static function register_font_collection( $config ) {
+		$new_collection = new WP_Font_Collection( $config );
+
+		if ( isset( self::$collections[ $config['id'] ] ) ) {
+			return new WP_Error( 'font_collection_registration_error', 'Font collection already registered.' );
```

Comment:
This needs to be translatable

Reply:
done here: https://github.com/WordPress/wordpress-develop/pull/5285/commits/f38bb5963b4a5a31bab31e8801f322988d43a2ce

Reply:
Confirming https://github.com/WordPress/wordpress-develop/commit/f38bb5963b4a5a31bab31e8801f322988d43a2ce does resolve the review 👍 

I'll mark this conversation as resolved, as leaving it expanded doesn't add or share contextual information for contributors.